docs/operate/get-started/other-hardware/create-module/_index.md

```diff
@@ -84,8 +84,17 @@ If you want to plan your module before you write it, you can use the following s
 3. **Decide on configuration attributes and dependencies**
 
    Make a list of required and optional attributes for users to configure when adding your module to a machine.
-   For example, you can require users to configure a path from which to access data, or a pin to which a device is wired, and you could allow them to optionally change a frequency from some default.
-   You'll need to add these attributes to the `Validate` and `Reconfigure` functions when you write the module.
+   Some examples of attributes:
+
+   - A filepath from which to access data
+   - A pin to which a device is wired
+   - An optional signal frequency to override a default value
+
+   You can also add dependencies to other resources that your module needs to use.
+   For example, if your module needs to access a camera, code your module to get a camera resource as a dependency.
+   This will allow you to access the camera with the camera API methods from within your module.
+
+   You'll add these attributes and dependencies to the `Validate` and `Reconfigure` functions when you write the module.
```

Comment:
I think the "code your module to get a resource as a dependency is not terribly helpful because we don't know how yet.
```suggestion
   For example, if your module needs to access a camera, add any required attributes and dependencies to the `Validate` and `Reconfigure` functions when you write the module.
   Then you can use the camera API methods from  within the module.
```

Reply:
I think that sounds as though you just add it in the config though

Reply:
Is that not what you do?

Reply:
I mean, this page is focused on writing the module, not on configuring it. And if the module you're writing needs access to a camera, right now the important thing that you as a reader need to focus on coding the module's validate and reconfigure methods to get the camera as a dependency. The configuration part is a much later step, and configuring something doesn't make it a dependency.